spec/models/post_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context 'method checking' do
+    it 'if data stores using new' do
+      post = Post.new({ title: 'mayur', body: 'good boy vaaf abvf av' }).save
+      expect(post).to be true
+    end
+
+    it 'if data stores using create' do
+      post = Post.create({ title: 'mayur', body: 'good boy vaaf abvf av' }).save
+      expect(post).to be true
+    end
+
+    it 'if first data deletes' do
+      post = Post.first.delete
+      expect(post).to be true
```

Comment:
I don't see any post record getting saved to database in this test case. how come you are deleting the record.